share/completions/brew.fish

```diff
@@ -6,10 +11,32 @@ function __fish_brew_needs_command
   return 1
 end
 
+function __fish_brew_needs_services_action
+  __fish_brew_using_command services
+
+  set cmd (commandline -opc)
+  if test (count $cmd) -le 2
+    return 0
+  end
+  return 1
+end
+
+function __fish_brew_services
+  brew services list | awk '{if (NR>1) print $1}'
```

Comment:
In our own scripts and completions we try to use `string` for these kinds of things.

This is just cutting after the first space, right? `string replace -r ' .*' ''` would work then.